infra/integration-image/Dockerfile

```diff
@@ -34,11 +34,23 @@ RUN curl -sL https://deb.nodesource.com/setup_7.x | bash - && \
     e2fsprogs \
     iptables \
     xfsprogs \
-    xz-utils && \
+    # Speed up ISO builds with already installed reqs
+    yum \
+    yum-utils \
+    cpio \
+    rpm \
+    ca-certificates \
+    xz-utils \
+    xorriso && \
 	# Cleanup
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+COPY *.repo /etc/yum/repos.d/
+RUN yum-config-manager --enable photon photon-updates
+
+RUN mkdir -p /build/rootfs
+RUN yum --installroot /build/rootfs install coreutils filesystem linux-esx e2fsprogs gzip haveged iproute2 iptables iputils logrotate net-tools openssh photon-release procps-ng sudo systemd tdnf vim systemd -y --nogpgcheck
```

Comment:
Speed is one thing, but there are other important reasons such as pinning the version and of course this (which is now causing all builds to master to fail this morning):
```
https://vmware.bintray.com/photon_release_1.0_x86_64/x86_64/gmp-6.0.0a-3.ph1.x86_64.rpm: [Errno 14] PYCURL ERROR 52 - "Empty reply from server"
Trying other mirror.
Error Downloading Packages:
  gmp-6.0.0a-3.ph1.x86_64: failure: x86_64/gmp-6.0.0a-3.ph1.x86_64.rpm from photon: [Errno 256] No more mirrors to try.
```